deepmd/pt/train/training.py

```
@@ -998,11 +1015,23 @@ def single_model_finetune(
             assert np.allclose(_data_stat_protect, _data_stat_protect[0]), (
                 "Model key 'data_stat_protect' must be the same in each branch when multitask!"
             )
+            share_kwargs = {
+                "model_key_prob_map": dict(zip(self.model_keys, self.model_prob)),
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)